src/Spryker/Shared/EventBehavior/EventBehaviorConstants.php

```diff
@@ -22,12 +22,20 @@ interface EventBehaviorConstants
 
     /**
      * Specification:
-     * - Chunck size for behavior events.
+     * - Chunk size for behavior events.
      *
      * @api
      */
     public const EVENT_BEHAVIOR_CHUNK_SIZE = 'EVENT_BEHAVIOR_CHUNK_SIZE';
 
+    /**
+     * Specification:
+     * - Chunk size for trigger events.
+     *
+     * @api
+     */
+    public const EVENT_TRIGGER_CHUNK_SIZE = 'EVENT_TRIGGER_CHUNK_SIZE';
```

Comment:
This constant should use %MODULE_NAME%:%CONSTANT_NAME% pattern
E.g. in this case value should be `EVENT_BEHAVIOR:EVENT_TRIGGER_CHUNK_SIZE`